tests/testsuite/cargo_add/detect_workspace_inherit_fuzzy/mod.rs

```diff
@@ -7,7 +7,6 @@ use cargo_test_support::str;
 use cargo_test_support::Project;
 
 #[cargo_test]
-#[ignore]
```

Comment:
Please update your PR so the tests are never ignored and the first two commits pass on their own.  THat will make it so this commit will then change their output, showing the change in behavior.

Reply:
Ok, I updated the PR.